modules/cnv-delete-vm-web.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-delete-vms.adoc
+
+[id="cnv-delete-vm-web_{context}"]
+
+= Deleting a virtual machine using the web console
+
+Deleting a virtual machine permanently removes it from the cluster.
+
+Delete a virtual machine using the &#8942; button of the virtual machine in the *Workloads* -> *Virtual Machines* list, or using the *Actions* control of the *Virtual Machine Details* screen.
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machines* from the side menu.
+. Click the &#8942; button of the virtual machine to delete and select *Delete Virtual Machine*. Alternatively, click the virtual machine name to open the *Virtual Machine Details* screen and click *Actions* -> *Delete Virtual Machine*.
```

Comment:
You can't jam options into one step like this. Update to use the same format as this step: https://github.com/openshift/openshift-docs/blame/master/modules/cnv-initiating-vm-migration-web.adoc#L19

Reply:
Thanks. I changed this to:

. Click the &#8942; button of the virtual machine to delete and select *Delete Virtual Machine*. 
** Alternatively, click the virtual machine name to open the *Virtual Machine Details* screen and click *Actions* -> *Delete Virtual Machine*.

FWIW I usually do not like giving users multiple options especially when the task is this simple, but maybe this is different in the open shift world. In the commercial world multiple options are frowned on and usually it's ultimately the UI that needs to make things plain to the user.